src/gatsby-ssr.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 
-exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
+// Gavin Johnson, 20220205: Changed over to an ES export so ES & CJS aren't mixed
+export function onRenderBody({ setHeadComponents }, pluginOptions) {
```

Comment:
Also changed from an "arrow function" to a regular function, which has potential implications but in this case I think it's fine.

Reply:
I changed this because I was experimenting with a custom build and the new version of Gatsby was throwing an error due to having mixed ES & CJS. It doesn't seem to have an issue with this plugin in practice, but I hit the error and figured it was a quick fix.

Reply:
Cool. Were you directly importing the code from this repository into your project?

I ask because the installation from the NPM repo should provide only a a version of this code that has been transpiled by Babel, and so you should not be seeing any "mixed" warnings/errors as everything is converted to CJS in those resulting files.

Regardless, fixing the mixed styles is no bueno...thanks!

Reply:
I was testing a modified version by importing it directly. Makes sense why it would never be an issue with the npm package.